cypress/integration/app/settingsflow.spec.ts

```diff
@@ -136,5 +136,7 @@ describe("Settings flow", () => {
     cy.get("#advanced-options").click();
 
     cy.findByLabelText(/host expiry window/i).should("have.value", "5");
+
+    cy.getHelloEmail();
```

Comment:
What if the helper was `getEmail()` (with the appropriate change to just return the response) and then this could be 

```suggestion
    cy.getHelloEmail().then((response) => {
    expect(response.status).to.eq(200);
    expect(response.body.items[0].To[0]).to.have.property("Domain");
    expect(response.body.items[0].To[0].Mailbox).to.equal("test");
    expect(response.body.items[0].To[0].Domain).to.equal("fleetdm.com");
    expect(response.body.items[0].Content.Headers.Subject[0]).to.equal(
      "Hello from Fleet"
    );
  });
```

Then we have a helper in commands.ts that grabs all the emails and we can use that helper in any test to look for the relevant email we need to work with.